### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get'

### ISSUE THREAD

**Issue by alitajelsir** [xuanruiqi/rebuild-initramfs-dracut-arch#1]
- I get the following error when trying to rebuild the initramfs:
	```
	Traceback (most recent call last):
	  File "/usr/bin/rebuild-initramfs", line 8, in <module>
	    sys.exit(main())
	             ^^^^^^
	  File "/usr/lib/python3.12/site-packages/rebuild_initramfs.py", line 198, in main
	    verbosity = 2 if config.get("verbose", False) else 1
	                     ^^^^^^^^^^
	AttributeError: 'NoneType' object has no attribute 'get'
	```
- `rebuild-initramfs-dracut` version `2.0.2-1` and `python` version `3.12.3`.

**Comment by xuanruiqi** [comment]
This is due to quirks in PyYAML: when encountered with an empty YAML object, it returns `None` instead of an empty object, as expected; if the YAML is malformed, it simply parses into a string. I've added some checks for these kinds of failure, and this bug should be gone in the new release.

**Comment by alitajelsir** [comment]
- Tried the new release `2.1.0` and got the following error:
	```
	Traceback (most recent call last):
	  File "/usr/bin/rebuild-initramfs", line 5, in <module>
	    from rebuild_initramfs import main
	  File "/usr/lib/python3.12/site-packages/rebuild_initramfs.py", line 5, in <module>
	    from numpy import isin
	ModuleNotFoundError: No module named 'numpy'
	```
- Sorry, I didn't know if I should open a new issue or not.

**Comment by xuanruiqi** [comment]
Please open a new issue for a new bug, but I don't use numpy. It must have been added by Copilot unbeknownst to me.

**Comment by xuanruiqi** [comment]
Fixed by c32d48aa81feb5cd15b9dfca866169532655b110.

**Comment by alitajelsir** [comment]
> Please open a new issue for a new bug
- Sorry for my ignorance, I didn't know if the current error is a continuation of the previous bug or not.
- Thanks for being helpful!.

**Comment by alitajelsir** [comment]
> Fixed by [c32d48a](https://github.com/xuanruiqi/rebuild-initramfs-dracut-arch/commit/c32d48aa81feb5cd15b9dfca866169532655b110).

Working fine now.
